spec/features/booking_spec.rb

```diff
@@ -0,0 +1,83 @@
+feature 'booking a place' do 
+  scenario 'I can book a place to stay' do 
+    sign_up
+    posting
+    click_link 'Main page'
+    click_link 'whatever'
+    click_link 'Book'
+    expect(page).to have_content ('user name your booking for whatever has been placed.')
+  end
+
+  scenario 'I can book a place to stay if I am signed in' do 
+    sign_up
+    posting
+    click_link 'Main page'
+    click_button 'Sign Out'
+    click_link 'whatever'
+    click_link 'Book'
+    expect(page).to have_content ('You need to log in or sign up to book.')
+  end
+
+  scenario 'I can see my booking requests' do
+    sign_up
+    posting
+    click_button 'Sign Out'
+    sign_up2
+    click_link 'whatever'
+    click_link 'Book'
+    expect(page).to have_content ('whatever')
+  end
+
+  scenario 'I can see requests made for my place' do 
+    sign_up
+    posting
+    click_button 'Sign Out'
+    sign_up2
+    click_link 'whatever'
+    click_link 'Book'
+    click_button 'Sign Out'
+    sign_in(email:'test@test.com', password:'password')
+    click_link 'My Listings'
+    click_link 'View My Bookings'
+    expect(page).to have_content ('whatever')
+  end
+  scenario 'I can confirm the booking request on my space' do 
+    sign_up
+    posting
+    click_button 'Sign Out'
+    sign_up2
+    click_link 'whatever'
+    click_link 'Book'
+    click_button 'Sign Out'
+    sign_in(email:'test@test.com', password:'password')
+    click_link 'My Listings'
+    click_link 'View My Bookings'
+    click_link 'Confirm'
+    expect(page).to have_content ('Booking confirmed')
+  end
+  scenario 'I can deny the booking request on my space' do 
+    sign_up
+    posting
+    click_button 'Sign Out'
+    sign_up2
```

Comment:
`sign_up2` doesn't tell me much about what this does - or how it differs from `sign_up1` (or `sign_up`?)